src/Npgsql/TypeMapping/INpgsqlTypeMapperExtensions.cs

```diff
@@ -14,34 +11,6 @@ namespace Npgsql;
 /// </summary>
 public static class INpgsqlTypeMapperExtensions
 {
-    /// <summary>
-    /// Sets up dynamic System.Text.Json mappings. This allows mapping arbitrary .NET types to PostgreSQL <c>json</c> and <c>jsonb</c>
-    /// types, as well as <see cref="JsonNode" /> and its derived types.
-    /// </summary>
-    /// <param name="mapper">The type mapper.</param>
-    /// <param name="serializerOptions">Options to customize JSON serialization and deserialization.</param>
-    /// <param name="jsonbClrTypes">
-    /// A list of CLR types to map to PostgreSQL <c>jsonb</c> (no need to specify <see cref="NpgsqlDbType.Jsonb" />).
-    /// </param>
-    /// <param name="jsonClrTypes">
-    /// A list of CLR types to map to PostgreSQL <c>json</c> (no need to specify <see cref="NpgsqlDbType.Json" />).
-    /// </param>
-    /// <remarks>
-    /// Due to the dynamic nature of these mappings, they are not compatible with NativeAOT or trimming.
-    /// </remarks>
-    [RequiresUnreferencedCode("Json serializer may perform reflection on trimmed types.")]
-    [RequiresDynamicCode("Serializing arbitrary types to json can require creating new generic types or methods, which requires creating code at runtime. This may not work when AOT compiling.")]
-    public static T EnableDynamicJson<T>(
```

Comment:
You think it's worth leaving EnableRecordsAsTuples and EnableUnmappedTypes here, or might as well just move those back to the builders?

Reply:
Let's say I was optimizing for time spent

Reply:
Would be nice to have consistency, but I can't say I really care

Reply:
Fixed with newfound motivation